### ISSUE TITLE
Loading Sidebar error on Edge

### ISSUE THREAD

**Issue by Pashaminkovsky** [pixiebrix/pixiebrix-extension#9324]
"loading sidebar" in the sidebar on Edge (version 130.0.2849.46 (Official Build) (64-bit)), <img width="539" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNnRrQnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--9dcc28a5db5f553adf54bed3d365ea58d9bc2193/image%20%287%29.png" alt="image (7).png" />

https://pixiebrix.slack.com/archives/C0436P48QHY/p1729610120708799

Affecting customer and individual user

**Comment by brittanyjoiner15** [comment]
And here's a bug report from community user for page editor: https://pixiebrix.zendesk.com/agent/tickets/769

**Comment by brittanyjoiner15** [comment]
@mnholtz the bus patrol ticket here: https://pixiebrix.zendesk.com/agent/tickets/767

**Comment by mnholtz** [comment]
Blocked due to inability to reproduce and insufficient information.

Information I still don't have:
- Confirmed emails of impacted users (still awaiting response from enterprise customer, and email for community user's PixieBrix acct is still unconfirmed)
- Reproduction steps
- Any errors in Datadog that seem to be correlative

Things I tried:
- Making assumptions about impacted emails & searching for errors in Datadog
- Hijacking presumed affected users on a resource-constrained Windows environment in Edge, and opening the sidebar

**Comment by brittanyjoiner15** [comment]
Community member is using version: `Version 130.0.2849.46 (Official build) (64-bit).`

Asking them to try on edge beta and see if that works. They said they are going to test that.

As far as reproduction, it seems like anytime they open the page editor they are seeing the loading page editor and no indication of the page editor loading. Should i ask them to take a screenshot of their console or anything? Let me know what else I can ask them to provide to help with troubleshooting. (Included user email in slack thread.)

**Comment by mnholtz** [comment]
> Should i ask them to take a screenshot of their console or anything? Let me know what else I can ask them to provide to help with troubleshooting. (Included user email in slack thread.)

If it's possible to get a view into console errors, that would be helpful. But it seems like this particular user might no longer be able to reproduce (per our slack conversation)

**Comment by Pashaminkovsky** [comment]
@mnholtz I'd suggest shelving this until we hear back from Buspatrol or other clients that they're experiencing this issue.

**Comment by mnholtz** [comment]
@Pashaminkovsky Yep, I agree. I'll defer to you whether or not you'd like to close or move this issue

**Comment by brittanyjoiner15** [comment]
@Pashaminkovsky and @mnholtz that user is still unable to view pixiebrix in the page editor though - they were only able to see it via edge beta. Im asking them for console screenshots, but otherwise do i need to let them know that they will need to use edge beta if they want to use PixieBrix?

**Comment by Pashaminkovsky** [comment]
@brittanyjoiner15 yeah if the user's found a workaround through Edge Beta let's tell them to use that. 

If we see this is issue come up again we can prioritize it, but for now it's only affecting one user who isn't a client / prospect so going down that rabbit hole isn't urgent or impactful

**Comment by brittanyjoiner15** [comment]
Sounds good!
